pyproject.toml

```diff
@@ -45,11 +44,18 @@ mypy = "^0.812"
 pylint = "^2.7.2"
 wheel = "^0.35.1"
 pylint-json2html = "^0.3.0"
-hyperscan = ">= 0.1.5"
 exrex = "^0.10.5"
 roman = "^3.3"
 pdoc3 = "^0.10.0"
 
+[tool.poetry.group.x86.dependencies]
+hyperscan = ">= 0.1.5"
```

Comment:
I don't think we should put this behind a `group`, because if we do, it means older poetry installations won't be able to install it, right?

Reply:
I think it depends on whether this is marked as optional or not.  If it isn't set to optional true, it should install automatically, I think.

Reply:
Interesting. This is worth checking just to be sure. pyproject.toml files have various metadata sections that can be used by lots of tools besides poetry, so I'd be a bit surprised if older poetry versions even read this section, since `tool.poetry.group` didn't exist as a section until recently. But I could be wrong!

Reply:
thats probably correct.  I hadn't considered it

Reply:
Only one way to find out. :)

Reply:
you are right. about older versions.